### PR TITLE
Tape recorders 2

### PR DESCRIPTION
**Pull request by stanalbatross**
# About the pull request

more opportunities for audio logs

# Explain why it's good for the game

atmosphere and soul

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: added a colony tape recorder, sprites by thwomper
add: added the colony tape recorder as a loadout item
add: added support + documentation for making custom audio log tapes
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/taperecorder.dm [hunk 1]
```diff
@@ -342,6 +328,37 @@
...
+
+/// do not use on maps
+/obj/item/device/taperecorder/colony/loadout
+	name = "\improper Weyland-Yutani brand tape recorder"
+	desc = "A cheap plastic tape recorder, mass produced by W-Y for distribution all over civilised space. To save on money, they cannot print transcripts of their tapes. You bought this one in a PX on Chinook before setting off."
+	starting_tape_type = /obj/item/tape/random/loadout
+	/// chance for it to spawn with no tape
+	var/spawn_empty_chance = 50
+	/// chance for it to spawn with a random LORE TAPE (if it does spawn with a tape) from the list below
+	var/spawn_lore_tape_chance = 20 // 10% chance
```
Comment (carlarctg) [inline code/game/objects/items/devices/taperecorder.dm:348]: erm 50% plus 20% isnt 10%..... its closer to 9.58462817482%......

File: code/game/objects/items/devices/taperecorder.dm [hunk 2]
```diff
@@ -489,12 +504,72 @@
...
+	End of recording.
+	Playback stopped.
+
+// have fun!
+
+*/
+
+/obj/item/tape/audio_log
+	name = "partially used tape" // RENAME!
+	desc = "A standard tape, made by the million in factories on Earth. This one has been partially used." // RENAME!
```
Comment (carlarctg) [inline code/game/objects/items/devices/taperecorder.dm:572]:
```suggestion
	desc = "A standard tape, made by the million in factories on Terra. This one has been partially used." // RENAME!
```

calling earth terra is soulful and reinforces the idea that it's some sort of legendary place marines and colonists look up towards from their shithole vessels and colonies (even if they're wrong)

File: code/modules/client/preferences_gear.dm [hunk 3]
```diff
@@ -680,3 +682,9 @@ var/global/list/gear_datums = list()
 	path = /obj/item/storage/box/pdt_kit
 	cost = 3
 	slot = WEAR_IN_BACK
+
+/datum/gear/tape_recorder
+	display_name = "WY Tape Recorder"
+	path = /obj/item/device/taperecorder/colony/loadout
+	cost = 4
```
Comment (carlarctg) [inline code/modules/client/preferences_gear.dm:689]:
```suggestion
	cost = 3
```

telling you from experience nobody will grab these if they're too costly

**Review by carlarctg — COMMENTED** [review]
coole

File: code/game/objects/items/devices/taperecorder.dm (hunk 2 above)
Comment (stanalbatross) [reply code/game/objects/items/devices/taperecorder.dm:572]: it's not called that anywhere else + we are not TGMC (TERRAgov marine corps)

**Comment by stanalbatross** [comment]
nvm can wait for tapes

**Review by harryob — APPROVED** [review]
code is fine

**Pull request closed without merging**